acceptance: TestDockerCLI failed

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2581222&tab=buildLog) on [release-20.2@3e61d3de4e7e0d21a45e67c11b063656f4fbd3fd](https://github.com/cockroachdb/cockroach/commits/3e61d3de4e7e0d21a45e67c11b063656f4fbd3fd):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (319.88s)
    test_log_scope.go:158: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI054980745
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_force_auth_log.tcl
    --- FAIL: TestDockerCLI/test_force_auth_log.tcl (2.85s)
        cli_test.go:89: non-zero exit code: 1

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cli/interactive_tests/test_force_auth_log.tcl
+++ b/pkg/cli/interactive_tests/test_force_auth_log.tcl
@@ -38,9 +38,9 @@ system "grep -q 'authentication failed' $logfile"
 
 end_test
 
-start_test "Check that the auth events have both node ID and cluster ID"
+start_test "Check that the auth events include the cluster ID"
 
-system "grep -q '\\\[n1,.*clusterID=........-....-....-....-............\\\] . authentication' $logfile"
+system "grep -q '\\\[n1,.*clusterID=........-....-....-....-............\\\] \[0-9\]* authentication' $logfile"
 
 end_test
 

